base_layer/wallet_ffi/.github/workflows/release.yml

```
@@ -0,0 +1,171 @@
+env:
+  CARGO_TERM_COLOR: always
+  TARI_TARGET_NETWORK: nextnet
+
+jobs:
+  build-wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)